test/e2e/service_options_test.go

```diff
@@ -72,6 +72,11 @@ func TestServiceOptions(t *testing.T) {
 	validateServiceMinScale(r, "svc2", "1")
 	validateServiceMaxScale(r, "svc2", "3")
 
+	t.Log("create and validate service with scale options ")
+	serviceCreateWithOptions(r, "svc2", "--scale", "5")
+	validateServiceMinScale(r, "svc2", "5")
+	validateServiceMaxScale(r, "svc2", "5")
+
 	t.Log("update and validate service with max scale option")
 	test.ServiceUpdate(r, "svc2", "--max-scale", "2")
```

Comment:
e2e failure:
service `svc2` exists (created at line no 73), try updating the service or create a fresh one after line:85

Reply:
Thank you for the help. I missed that.